modules/http/src/main/java/org/mule/module/http/internal/request/client/HttpRequesterBuilder.java

```diff
@@ -162,6 +162,10 @@ else if (httpRequester.getConfig() == null)
                     }
                 }
             }
+            else
+            {
+                requestConfig.start();
```

Comment:
Is there a chance you'll start it several times?

Reply:
This scenario is covered by the start method code:

https://github.com/mulesoft/mule/blob/mule-3.x/modules/http/src/main/java/org/mule/module/http/internal/request/DefaultHttpRequesterConfig.java#L340